### PR TITLE
Add filter to prevent empty image tags from rendering in figures

### PR DESCRIPTION
**Pull request by Tetrino**
Previously if the image source was missing the tag would still render.
This was fine in many browsers but caused a big empty space in IE.
This pull request should fix that issue.

Zendesk: https://govuk.zendesk.com/agent/tickets/4391436

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/views/components/_figure.html.erb [hunk 1]
```diff
@@ -4,18 +4,19 @@
   credit ||= ''
 %>
 <figure class="app-c-figure" lang="en">
-  <img class="app-c-figure__image" src="<%= src %>" alt="<%= alt %>">
-    <figcaption class="app-c-figure__figcaption">
-      <% unless caption.blank? %>
-        <p class="app-c-figure__figcaption-text">
-          <%= caption %>
+  <% unless src.blank? %>
```
Comment (brucebolt) [inline app/views/components/_figure.html.erb:7]: It's a really minor nitpick, but I know our linter prefers `if src.present?` over `unless src.blank?`.  We don't lint our views, but it's probably good practice to do that here too.

Comment (brucebolt) [reply app/views/components/_figure.html.erb:7]: On the same point, I know you didn't write that code, but `unless caption.blank?` below should also be changed to `if caption.present?`.

**Review by brucebolt — COMMENTED** [review]
Looks good.  I've left a really minor nitpick comment that should be a very minor change.

File: app/views/components/_figure.html.erb (hunk 1 above)
Comment (Tetrino) [reply app/views/components/_figure.html.erb]: Good shout. Updated.

**Review by brucebolt — APPROVED** [review]

**Pull request merged**